util/dockerfiles/github-ci/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# This is a container used to run CI jobs on GH
+FROM ubuntu:22.04
```

Comment:
Currently, we just use the `ubuntu-latest` alias to use github actions newest ubuntu container. Is there a way to do something similar here so we don't have to remember to bump our versions?

Reply:
I opted for 22.04 to get llvm-13 available. If you do `FROM ubuntu:latest`, you'll get 20.04 today which only has llvm-12

Reply:
Gotcha, can you add a comment along that lines.